ConferenceV2/app/services/singletons/conference_call_manager.py

```
@@ -28,7 +30,7 @@ def __init__(
         self.conferences: Dict[str, ConferenceCall] = {}
         self.ws_base_url = os.environ.get("WS_SERVER_EP", "")
 
-    def create_conference(self, teacher_phone: str, student_phones: List[str], leader_phone: str = None, teacher_name: str | None = None, student_names: List[str] | None = None) -> ConferenceCall:
+    async def create_conference(self, teacher_phone: str, student_phones: List[str], leader_phone: str = None, teacher_name: str | None = None, student_names: List[str] | None = None) -> ConferenceCall:
```

Ruff (0.15.12)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)